### PR TITLE
Balisong

### PR DESCRIPTION
**Pull request by Kmc2000**
Adds a balisong (butterfly knife) to the game for traitors, it costs 4 tc and does 21 damage when deployed. It is concealable and only clicks when deployed, thus it serves as a more quiet esword or edagger.

It is under the "Stealthy Weapons" tab of the uplink

https://i.gyazo.com/763f54ffd7938d11041bac1f79f9d07d.mp4 (twirling animation)

What does it do?

It sits halfway between an energy sword and a normal knife, you twirl it around and do tricks when it's deployed and is hard to detect when not deployed, the only sound it makes is a normal click so it's not as obvious/infamous as an energy sword.

##### Changelog

:cl:
rscadd: Adds in a butterfly knife for traitors, watch out for spies!

/:cl:

### CODE REVIEW

**Comment by matskuman5** [comment]
>4 tc and does 21 damage

>It is concealable

>the only sound it makes is a normal click

beautiful

**Comment by Kmc2000** [comment]
>mat

The edagger does 18 for 3 with the swoosh noise.

This is not unreasonable

**Comment by Super3222** [comment]
Sounds fun, it could even replace edaggers in the future.

**Comment by Cruix** [comment]
How is this different from a switchblade?

**Comment by Kmc2000** [comment]
>cruix

Higher damage, spinning animation, badassery.

**Comment by Super3222** [comment]
It could use something a bit more... unique? No?

**Comment by Kmc2000** [comment]
>super

Like...? what exactly?

**Comment by Super3222** [comment]
Make it do 30 damage when you attack someone facing the other direction? Normal melee could be reduced to 18 (e dagger). Effectively replacing it.

You should consider letting this fit in ears slots, or at least the case to holster the knife.

**Comment by drderp3635** [comment]
> You should consider letting this fit in ears slots, or at least the case to holster the knife.

im not sure if you know what a balisong is, but you wouldn't get one behind your ear comfortably

**Comment by ShadowDeath6** [comment]
not a big fan of the red decals tbh, regular white knife would look better

**Comment by RoboticVodka** [comment]
Shadow I think that's blood... Not decals...

**Comment by Kmc2000** [comment]
>shadow

He's correct those are part of the design

**Comment by ghost** [comment]
This is just a better edagger.

The question is why?

**Comment by ShadowDeath6** [comment]
Because outdated and obsolete metal knives > beams made of hardlight, obviously

**Comment by Keekenox** [comment]
Add backstab and less CSGO meme sprite and you will have my vote. 

Or made backstabs always stun

File: code/game/objects/items/weapons/melee/misc.dm [hunk 1]
```diff
@@ -222,3 +222,59 @@
...
+		w_class = 4 //don't want the blade poking a hole in your bag when open
+		force = 21 //lower than esword because you can hide this without the SWIIIINCHHH esword noise
+		hitsound = 'sound/weapons/bladeslice.ogg'
+		attack_verb = list("slit", "sliced", "eviscerated", "attacked")
+
+	else
+		user << "<span class ='notice'>You flick the blade back into its handle like a badass.</span>"
+		icon_state = "balisong_closed"
+		item_state = null //no sprite for concealment because it's stealthy
+		slot_flags = SLOT_BELT //beltisong
```
Comment (Time-Green) [inline code/game/objects/items/weapons/melee/misc.dm:264]: Why's this here?

**Comment by ShadowDeath6** [comment]
Oh it's based off of csgo shit no wonder I don't like the sprite

File: code/game/objects/items/weapons/melee/misc.dm (hunk 1 above)
Comment (ShadowDeath6) [reply code/game/objects/items/weapons/melee/misc.dm:264]: Meme comment

**Comment by Keekenox** [comment]
Kmc, I suggest closing this and rethinking it's gameplay a little bit, I like this, but I like what it could be even more.

File: code/game/objects/items/weapons/melee/misc.dm (hunk 1 above)
Comment (Time-Green) [reply code/game/objects/items/weapons/melee/misc.dm:264]: No, I mean. He set the slot to belt, but why? It was already set to belt, this does absolutely nothing

**Comment by ghost** [comment]
or just make the edagger do whatever shit you wanted this to do

File: code/game/objects/items/weapons/melee/misc.dm (hunk 1 above)
Comment (Time-Green) [inline code/game/objects/items/weapons/melee/misc.dm:264]: Remove this. It's never actually changed, so this is pointless.

**Review by Time-Green — CHANGES_REQUESTED** [review]

**Comment by ShadowDeath6** [comment]
Fix your shit pham

**Comment by Cruix** [comment]
There is no reason to have this in the uplink when we already have a weapon that fills exactly the same niche. Remove it from the uplink and we can put it in gang or something later.

**Comment by Mrcreepssss** [comment]
i remember that on a server i used to play on you could craft something similair to this, it used plasteel and metal. don't know the stats though, maybe that's an idea?

**Pull request closed without merging**